packages/component/src/Utils/FocusRedirector.js

```diff
@@ -0,0 +1,28 @@
+import PropTypes from 'prop-types';
+import React, { useCallback } from 'react';
+
+const FocusRedirector = ({ className, onFocus, redirectRef }) => {
```

Comment:
can you explain this util for me?

Reply:
`focusTrap` to send focus away to specific element

Reply:
plz add comment

Reply:
```js
// This is an element, when focused, will send the focus to the ref specified in "redirectRef".
// Although the focus is being redirected, browser will scroll this redirector element into view.

// Browser's "scrollIntoView()" call cannot be prevented through event.preventDefault() in both
// bubble and capture phase of "focus" event.

// When this focus redirector is put inside a scrollable container, you may want to resize or reposition
// it to prevent unintentional scroll done by the browser default behavior.
```